torchft/diloco_trainer.py

```diff
@@ -0,0 +1,310 @@
+import copy
```

Comment:
Does this makes sense under torchft/? Should we add an underscore to indicate it's internal or move it under some `testing` dir or something?

Just don't want folks getting confused and importing this directly for whatever reason

Reply:
agree with this